README.md

```diff
@@ -19,6 +19,22 @@ SYNOPSIS
       a folder in a bag:     'id:ef425828-e4ae-4d58-bf6a-c89cd46df61c/data/files/Documents/*'
 
 
+### HTTP service
+
+When started with the sub-command `run-service` an HTTP API becomes available summarized in the following table.
+"Method" refers to the HTTP method used in the request. "Path" is the path pattern used.
+In a path pattern `*` refers to any completion of the path, and placeholders for variables start with a colon.
+Optional parts are enclosed in square brackets, a pipe symbol denotes alternatives and parentheses are used to group those alternatives.
+
+Method   | Path                                |Action
+---------|-------------------------------------|------------------------------------
+`GET`    | `/fileindex`                        | Return a simple message to indicate that the service is up: "EASY File index is running."
+`POST`   | `/fileindex/init[/:store]`          | Index all bag stores or just one. Eventual obsolete items are cleared.
+`POST`   | `/fileindex/update/:store[/:uuid]`  | Index all files of one bag. Eventual obsolete file items are cleared.
+`DELETE` | `/fileindex/:store[/:uuid]`         | Remove all items or the items of a store or bag.
+`DELETE` | `/fileindex?q=*`                    | Remove the items matching the specified solr query.
```

Comment:
make is slightly more clear that the '*' in this line is the sole query 'string'. 
For instance if we make the comment: Remove the items matching the specified solr query (*)